### DISCUSSION TITLE
How to distruct a sink started with a2dp_sink.start("NAME")

### DISCUSSION THREAD

**Discussion by FrankHovis** [pschatzmann/ESP32-A2DP#213] [General]
I want to close the sink connection when nothing is played without simply shutting down the CPU as the example shows. What's the best way to close the connection?

**Comment by clarkbremer** [comment]
Just call `a2dp_sink.disconnect()`

**Comment by FrankHovis** [comment]
Ok thanks, that works. Unfortunately I've just found that Bluetooth and WiFi won't work together so I'm going to have to think again.
